test/images/dnsutils/Dockerfile

```diff
@@ -17,3 +17,6 @@ FROM BASEIMAGE
 CROSS_BUILD_COPY qemu-QEMUARCH-static /usr/bin/
 
 RUN apk add --no-cache bind-tools dnsmasq
+
+ADD https://github.com/coredns/coredns/releases/download/v1.5.0/coredns_1.5.0_linux_BASEARCH.tgz /coredns.tgz
```

Comment:
does this work for arm, mips, ppc64le and other non x86 architectures?

Reply:
Yes, that was the point. as you can see in the URL, there's a ``BASEARCH``, which is then replaced accordingly by the ``image-util.sh`` build script. So, the arm image will have the coredns_1.5.0_linux_arm binary, and so on.

I've created a few containers using those images with different archs and tried executing ``/coredns``:

    ubuntu@ubuntu:~/kubernetes/test/images$ docker exec test-arm /coredns --help
    Usage of /coredns:
      -conf string
            Corefile to load (default "Corefile")
      -cpu string
            CPU cap (default "100%")
      -dns.port string
            Default port (default "53")
      -pidfile string
            Path to write pid file
      -plugins
            List installed plugins
      -quiet
            Quiet mode (no initialization output)
      -version
            Show version
    ubuntu@ubuntu:~/kubernetes/test/images$ docker exec test-arm64 /coredns --help
    Usage of /coredns:
      -conf string
            Corefile to load (default "Corefile")
      -cpu string
            CPU cap (default "100%")
      -dns.port string
            Default port (default "53")
      -pidfile string
            Path to write pid file
      -plugins
            List installed plugins
      -quiet
            Quiet mode (no initialization output)
      -version
            Show version
    ubuntu@ubuntu:~/kubernetes/test/images$ docker exec test-ppc64le /coredns --help
    Usage of /coredns:
      -conf string
            Corefile to load (default "Corefile")
      -cpu string
            CPU cap (default "100%")
      -dns.port string
            Default port (default "53")
      -pidfile string
            Path to write pid file
      -plugins
            List installed plugins
      -quiet
            Quiet mode (no initialization output)
      -version
            Show version